nuitka/PythonFlavors.py

```diff
@@ -220,6 +220,19 @@ def isUninstalledPython():
     return isAnacondaPython() or "WinPython" in sys.version
 
 
+def isRelocatable():
```

Comment:
I think I have called these things "uninstalled". And for them, a shell script is created to execute them as of recently, this strives to also persist the PYTHONPATH at compile time, PYTHONHOME, and what not.

Setting an rpath to find the libpython for sure, is a good idea anyway. And actually maybe independent of the fact if it's a system installed Python or not.

As for `sysconfig.get_config_var("PYTHON_BUILD_STANDALONE")` I am starting to feel, that this is a Python flavor, and UV Python actually is not, but might be producing CPython or Standalone Python (I need to check for proper naming), and that I should have that as the flavor instead.

Reply:
Check `isUninstalledPython` for the current uses.